### PR TITLE
Added test for maintaining references after copying

### PR DESCRIPTION
**Pull request by dantleech**
Test to expose bug reported in https://github.com/jackalope/jackalope-doctrine-dbal/issues/251

### CODE REVIEW

File: tests/10_Writing/CopyMethodsTest.php [hunk 1]
```diff
@@ -186,16 +186,42 @@ public function testCopyNoUpdateOnCopy()
...
+     */
+    public function testCopyUpdateReferencesSingleValue()
     {
-        $src = '/tests_write_manipulation_copy/testCopyUpdateOnCopy/srcNode';
-        $dst = '/tests_write_manipulation_copy/testCopyUpdateOnCopy/dstNode/srcNode';
+        $src = '/tests_write_manipulation_copy/testCopyUpdateReferrersSingleValue/srcNode';
+        $dst = '/tests_write_manipulation_copy/testCopyUpdateReferrersSingleValue/dstNode';
+        $ref = '/tests_write_manipulation_copy/testCopyUpdateReferrersSingleValue/referencedNode';
+
         $this->ws->copy($src, $dst);
```
Comment (dbu) [inline tests/10_Writing/CopyMethodsTest.php]: we should have the assertions before and after refreshing, to make sure both the object state and the storage state are correctly updated.

**Comment by dbu** [comment]
@dantleech can you please update this test to assert both before and after refreshing to be sure both memory state and database get updated?

**Comment by dantleech** [comment]
Updated.

**Comment by dbu** [comment]
thanks!

**Pull request merged**